### PR TITLE
updated the  actions

### PR DESCRIPTION
**Pull request by rishav-sumo-dev**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/platform-services/automation-service/app-central/integrations/virustotal.md [hunk 1]
```diff
@@ -7,25 +7,26 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-* **Scan File** (*Enrichment*) - Gather information on a specific file.
-* **Scan URL** (*Enrichment*) - Gather information on a specific URL.
+* **Scan File** (*Enrichment*) - Scan the specific file.
+* **Scan URL** (*Enrichment*) - Scan on a specific URL.
 * **URL Reputation** (*Enrichment*) - Gather reputation information for multiple URLs.
-* **File Scan** **V2** (*Enrichment*) - Acquire information about a specific file without saving the report.
-* **URL Scan V2** (*Enrichment*) - Obtain information about a particular URL without storing the report.
-* **Search VirusTotal** (*Enrichment*) - Search for File Hash, URL, Domain, IP address and comments by tags.
+  <br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/virus-total/virus-total-1.png')} style={{border:'1px solid gray'}} alt="virus-total" width="800"/>
+  <br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/virus-total/virus-total-2.png')} style={{border:'1px solid gray'}} alt="virus-total" width="800"/>
```
Comment (jpipkin1) [inline docs/platform-services/automation-service/app-central/integrations/virustotal.md:29]: @rishav-sumo-dev, these two images are the same.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@rishav-sumo-dev, I made some minor updates (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4267/commits/bdb1b4c93ad1c4121481628bd66c84853c0f830a)").  Did you mean to make both those images the same?

File: docs/platform-services/automation-service/app-central/integrations/virustotal.md (hunk 1 above)
Comment (rishav-sumo-dev) [reply docs/platform-services/automation-service/app-central/integrations/virustotal.md:29]: Nope they different one is `Scan Url` & `Scan File`
Screenshot 2024-07-03 at 9 29 54 PM

Comment (jpipkin1) [reply docs/platform-services/automation-service/app-central/integrations/virustotal.md:29]: Oh, I see! Thank you.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**